src/main/index.ts

```diff
@@ -76,3 +83,74 @@ ipcMain.handle('get-locale', () => {
 
   return lang
 })
+
+const menuTemplate = [
```

Comment:
do we need translations for these?

Reply:
yep, added

Reply:
@achou11 if you could look at how I've implemented menu translation before I merge pls? :)

Reply:
doesn't work as implemented because you're trying to use `react-intl` in a Node process. You need to import the relevant functions from `@formatjs/intl` (see https://formatjs.io/docs/getting-started/installation) e.g.

```js
import {createIntl, createIntlCache} from '@formatjs/intl'
```

Reply:
are you sure it's not working? Works without issue for me...

Reply:
after debugging, no issue with implementation as I thought. Was just me not building the translations file before running the app (was running `npm run dev` instead of `npm start`)